test/unit/legacy_wrappers/gridfs.test.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const {
+  MongoClient,
+  GridFSBucket,
+  FindCursor,
+  GridFSBucketWriteStream
+} = require('../../../src/index');
+const mongodbDriver = require('mongodb');
+const sinon = require('sinon');
+const { expect } = require('chai');
+const { toLegacy } = require('../../../src/utils');
+
+const iLoveJs = 'mongodb://iLoveJavascript';
+
+describe('legacy_wrappers/gridfs.js', () => {
+  let client;
+  let bucket;
+  beforeEach(async function () {
+    client = new MongoClient(iLoveJs);
+    bucket = new GridFSBucket(client.db());
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+    await client.close();
+  });
+
+  it('should convert bucket.find(filter, options) to legacy version', () => {
+    const spy = sinon.spy(mongodbDriver.GridFSBucket.prototype, 'find');
+    const find = bucket.find({ filter: 1 }, { options: true });
+    expect(find).to.be.instanceOf(FindCursor);
+    expect(spy).to.be.calledWithExactly({ filter: 1 }, { options: true });
+  });
+
+  // We do not have an API that returns a bucket, users just use the constructor
+  // however, it may be useful to have this exist for a quick way to fix inconsistent instances
+  it('should define a toLegacy helper', () => {
+    expect(bucket[toLegacy]()).to.be.instanceOf(GridFSBucket);
+  });
+
+  it('should return legacy GridFSBucketWriteStream from openUploadStream', () => {
+    expect(bucket.openUploadStream('filename')).to.be.instanceOf(GridFSBucketWriteStream);
+  });
```

Comment:
similar request here as Bailey made earlier, can we clearly indicate that we are using a legacy version here, e.g., by not destructuring the imports?

Reply:
done!